src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1332,14 +1334,14 @@ public void LanguageVersionAdded_Canary()
             // - update the "UpgradeProject" codefixer
```

Comment:
Yes, I realized that too. But I came up with a solution, by leveraging another feature that's useful for point releases, the `#version` directive.

I have a PR almost ready: https://github.com/dotnet/roslyn/pull/17957